docs/source/migration.rst

```diff
@@ -70,7 +67,18 @@ Or to declare separate values for ``fit`` and ``predict``:
 
 .. code:: python
 
-   clf = KerasClassifier(fit__batch_size=32, predict__batch_size=10000)
+   clf = KerasClassifier(..., fit__batch_size=32, predict__batch_size=10000)
+
+If you want to change the parameters on a live instance, you can do:
+
+.. code:: python
+
+   clf = KerasClassifier(...)
+   clf.set_params(fit__batch_size=32, predict__batch_size=10000)
+   clf.fit(...)
+
+This effectively gives you the same result as passing ``**kwargs``, just with
```

Comment:
```suggestion
Functionally, this is the same as passing these parameters during initialization, just with
```

Reply:
I am specifically trying to give users a way that they can do this if they do _not_ want to pass the parameters via initialization, hence the comparison to passing `**kwargs`. The first part looks good though.

Reply:
I'm confused by your statement. I only tried to replace "`**kwargs`" with words. Wouldn't you pass `**kwargs` to `__init__`?

Reply:
Sorry if it was confusing. What I meant is that this should be the same as passing `**kwargs` to `fit`. So combining my thoughts and yours, I think it would look something like:
```
Functionally, this is the same as passing these parameters to ``fit``, just with one more function call.
```

Reply:
> Functionally, this is the same as passing these parameters to ``fit``, just with one more function call.

Yeah, I think that's a good representation of the message you were trying to convey. I might add a couple words about how it's better because it follows the Sciikit-learn API.

Reply:
Edited to:

```
Functionally, this is the same as passing these parameters to ``fit``, just with one more function call.
This is much more compatible with the Scikit-Learn API.
In fact, this is what Scikti-Learn does in the background for hyperparameter tuning.
```